### PR TITLE
update json serializer contentType

### PR DESCRIPTION
**Pull request by Sufiane**
permit content-type header to be either lower cased or UPPER cased.

### CODE REVIEW

**Review by remmmmmm — APPROVED** [review]

File: lib/paypalhttp/serializer/json.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Json {
   }
 
   contentType() {
-    return /^application\/json/;
+    return /^application\/(json|JSON)/;
```
Comment (skratchdot) [inline lib/paypalhttp/serializer/json.js:13]: why not just make the regex case insensitive?
```
return /^application\/json/i;
```

**Review by skratchdot — CHANGES_REQUESTED** [review]

File: lib/paypalhttp/serializer/json.js (hunk 1 above)
Comment (Sufiane) [reply lib/paypalhttp/serializer/json.js:13]: you're right it could be better to do so

**Review by tfang17 — APPROVED** [review]

**Comment by Sufiane** [comment]
The error happened again in production for us....

Could it be possible to merge this ASAP ?

**Comment by tfang17** [comment]
Need this merged ASAP!

**Comment by ganeshramc** [comment]
Closing this based on this: https://github.com/paypal/paypalhttp_node/pull/9

**Pull request closed without merging**